pkgs/servers/softether/4.29.nix

```
@@ -0,0 +1,228 @@
+{ stdenv, fetchurl
+, openssl, readline, ncurses, zlib
+, dataDir ? "/var/lib/softether" }:
+
+stdenv.mkDerivation rec {
+  name = "softether-${version}";
+  version = "4.29";
+  build = "9678";
```

```suggestion
  build = "9680";
```
now published upstream